public/css/stylesheet.css

```
@@ -676,3 +676,10 @@ img.mana-symbol {
 	vertical-align: top;
 	line-height: 0.9;	
 }
+
+@media (min-width: 768px) {
+	.sevenCol {
```

Can we name this consistently with the bootstrap columns?
```suggestion
	.col-seven {
```